src/humanize.cr

```diff
@@ -152,17 +152,17 @@ struct Number
   # delimiter (see `#format`).
   #
   # See `Int#humanize_bytes` to format a file size.
-  def humanize(io : IO, precision = 3, separator = '.', delimiter = ',', *, base = 10 ** 3, significant = true, prefixes : Indexable = SI_PREFIXES) : Nil
-    humanize(io, precision, separator, delimiter, base: base, significant: significant) do |magnitude, _|
+  def humanize(io : IO, precision = 3, separator = '.', delimiter = ',', *, base = 10 ** 3, significant = true, unit_separator = nil, prefixes : Indexable = SI_PREFIXES) : Nil
```

Comment:
I agree, documentation is needed.

We also have an opportunity to mention:

"Consider using `'\u00A0'` (non -breaking space) as the value for `unit_separator`, it can prevent the output from being split across lines".

Reply:
For sure - I'm not immediately seeing where I can add to the docs, can someone point me in the right direction? 😅